src/lib/viewers/controls/color-picker/__tests__/ColorPickerControl-test.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { AnnotationMode } from '../../annotations/types';
+import ColorPickerControl from '../ColorPickerControl';
+
+describe('ColorPickerControl', () => {
+    const onAnnotationColorClick = jest.fn();
+
+    const getWrapper = (props = {}): ShallowWrapper =>
+        shallow(<ColorPickerControl onAnnotationColorClick={onAnnotationColorClick} {...props} />);
+
+    const getToggleButton = (wrapper: ShallowWrapper): ShallowWrapper => wrapper.find('.bp-ColorPickerControl-button');
+
+    describe('render', () => {
+        test('should render null if annotationMode is not AnnotationMode.DRAWING', () => {
+            const wrapper = getWrapper({ annotationMode: AnnotationMode.REGION });
+
+            expect(wrapper.find('.bp-ColorPickerControl').length).toBe(0);
```

Comment:
Let's use `.exists` rather than checking the length. We also have `.isEmptyRender` for null cases.